server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -56,7 +56,7 @@
   private static final String ID_MATCHER =
       "([a-zA-Z0-9/\"[ ],:@#$%&{}()*=?!~\\[\\]\\-+\\u2E80-\\u9FFF_]+)";
 
-  private static final String STORAGE_GROUP_MATCHER = "([a-zA-Z0-9_.\\u2E80-\\u9FFF]+)";
+  private static final String STORAGE_GROUP_MATCHER = "([a-zA-Z0-9_.\\-\\u2E80-\\u9FFF]+)";
 
```

Comment:
Update the user guide?

Reply:
Hi, there is another document need to be update as well...
https://github.com/apache/iotdb/blob/master/docs/UserGuide/Data-Concept/Data-Model-and-Terminology.md

Reply:
Okay, thx for reminding